src/utils/markets/marketEmission.ts

```diff
@@ -22,15 +22,15 @@ export const computeMarketsEmission = (
   const total = totalBorrowUSD.add(totalSupplyUSD).div(WAD);
 
   const marketsEmissions: {
-    [market: string]: MarketEmission | undefined;
+    [market: string]: Optional<MarketEmission>;
   } = {};
   let marketEmissionTotal = BigNumber.from(0);
   Object.keys(ageOneMarketsParameters).forEach((marketAddress) => {
-    const market: Market = ageOneMarketsParameters[marketAddress];
+    const market: MarketMinimal = ageOneMarketsParameters[marketAddress];
     // total market value at the beginning of the age
     const totalMarketUSD = market.totalBorrow.add(market.totalSupply).mul(market.price); // 18 * 2 units
     const marketEmission = totalMarketUSD.mul(totalEmission).div(total); // in WEI units
-    const supplyTokens = marketEmission.mul(market.p2pIndexCursor).div(BASE_UNITS);
+    const supplyTokens = marketEmission.mul(market.p2pIndexCursor).div(BASIS_POINTS);
```

Comment:
Oh, we don't have PercentageMath implemented. Perhaps in another PR: can you create an issue if you don't want to implement it now?

Reply:
Let's create an issue!